auth/src/main/res/layout/fui_welcome_back_email_link_prompt_layout.xml

```diff
@@ -50,10 +50,11 @@
                 android:id="@+id/email_footer_tos_and_pp_text"
                 style="@style/FirebaseUI.PrivacyFooter"
                 app:layout_constraintTop_toBottomOf="@+id/button_sign_in"
-                app:layout_constraintBottom_toBottomOf="parent" />
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintEnd_toEndOf="parent" />
```

Comment:
Hmmm, is this intentional? I think I remember this view being `match_parent`, but if not it'll get stuck to the right.